projects/dev.py

```diff
@@ -31,9 +31,14 @@
     'django.contrib.admin',
     'django.contrib.admindocs',
 
-    # Our own apps
-    'openedx_learning.apps.core.publish.apps.PublishConfig',
+    # Learning Core Apps
     'openedx_learning.apps.core.compose.apps.ComposeConfig',
+    'openedx_learning.apps.core.itemstore.apps.ItemStoreConfig',
+    'openedx_learning.apps.core.publish.apps.PublishConfig',
```

Comment:
> The `apps` namespace package seems unnecessary and out-of-step with other edx-platform libraries. Why not just `openedx_learning.core.itemstore`?

Mostly because I wanted a place to hang `lib` for things like the field types.

> I notice that you are using verbs for some of the core app names. Wouldn't it be more standard to use nouns? (`composition`, `publishing`)

Yeah, I was going for the verb form for a while before because it was shorter, before I landed on `itemstore` for one of them. I'll switch it back over to nouns.

Reply:
> Mostly because I wanted a place to hang lib for things like the field types.

The 2nd-level packages could be `openedx_learning.core`, `openedx_learning.lib`, and `openedx_learning.contrib`.

I remember as a new dev being frazzled by the long import paths of `openedx.core.djangoapps...` in edx-platform. I could never remember the paths; I was always copying and pasting, and it made the platform feel more overwhelming.

Reply:
Okay, I'll give that a shot. I do like not having `apps` appear twice in that path (package + apps.py). Thank you!